### PR TITLE
Add publisher config to deployment yaml

### PR DESCRIPTION
**Pull request by seswarrajan**
Signed-off-by: Eswar Rajan Subramanian <eswar@accuknox.com>

### CODE REVIEW

File: deployments/k8s/deployment.yaml [hunk 1]
```diff
@@ -69,7 +69,12 @@ data:
       cron-job-time-interval: "0h0m10s"         # format: XhYmZs
       dbname: ./accuknox-obs.db
       system-observability: true
-      network-observability: true
+      network-observability: false
```
Comment (Ankurk99) [inline deployments/k8s/deployment.yaml]: why are we disabling network-observability?

Comment (seswarrajan) [reply deployments/k8s/deployment.yaml]: @Ankurk99 We do not have network observability. We will be using kubearmor to get network observability and generic network logs + observability as well.

Comment (rajaSahil) [reply deployments/k8s/deployment.yaml]: @seswarrajan  `network-observability` is not being used anywhere in the code?

Comment (Ankurk99) [reply deployments/k8s/deployment.yaml]: Ah! so have we removed the cilium network observability support?

Comment (seswarrajan) [reply deployments/k8s/deployment.yaml]: We are storing logs into network_logs DB and also retreiving it for observability. But not being used. 
Which will be removed in future.

**Review by Ankurk99 — APPROVED** [review]
LGTM

**Pull request closed without merging**